clang-tools-extra/test/clang-tidy/checkers/readability/identifier-naming-default-lower.cpp

```diff
@@ -0,0 +1,67 @@
+// RUN: %check_clang_tidy %s readability-identifier-naming %t -- \
+// RUN:   -config='{CheckOptions: { \
+// RUN:     readability-identifier-naming.DefaultCase: "lower_case", \
+// RUN:   }}'
+
+// DefaultCase enables every type of identifier to be checked with same case
+#define MyMacro
+// CHECK-MESSAGES: :[[@LINE-1]]:9: warning: invalid case style for default 'MyMacro' [readability-identifier-naming]
```

Comment:
This test fails, as macro checking appears to be handled separately. The only mention of `SK_MacroDefinition` is in `getMacroFailureInfo` which does not appear to be used anywhere?

Reply:
Okay, it was actually used as polymorphic override, it was helpful to run it under `gdb` to understand more of the code. Handling case where default case is defined but macro case is not was sufficient.